src/group/controller/collection.ts

```diff
@@ -78,11 +78,13 @@ class GroupMemberCollectionController extends Controller {
       throw new UnprocessableEntity('The "userHref" property must be set, and must point to the member you are adding to the group.');
     }
 
+
     const newMemberHref = ctx.request.body.userHref;
     let newMember;
 
     try {
-      newMember = await userService.findByHref(newMemberHref);
+      newMember = await userService.getByHref(newMemberHref);
+      newMember = await userService.findByHref(newMember);
```

Comment:
The logic of getByHref should be part of findByHref. It's also not great to re-use variables for different purposes.

But the idea is if i call `findByHref`, I expect to be able to to pass a url (absolute or relative).